google/resource_cloudfunctions_function.go

```diff
@@ -225,6 +227,12 @@ func resourceCloudFunctionsCreate(d *schema.ResourceData, meta interface{}) erro
 	if err != nil {
 		return err
 	}
+	// We do this extra validation here since most regions are not valid, and the
```

Comment:
I see, the provider-level region skips the field validation. May be add a mention about this here?